content/telegraf/v1.19/external_plugins/write_external_plugin.md

```diff
@@ -0,0 +1,56 @@
+---
+title: External plugins
+description:
+menu:
+  telegraf_1_19:
+     name: External plugins
+     weight: 50
+     parent: External plugins
+---
+Set up your plugin to use it with `execd`.
+
+Follow the guidelines for our general [input](/docs/INPUTS.md),
+[output](/docs/OUTPUTS.md), [processor](/docs/PROCESSORS.md), and [aggregator](/docs/AGGREGATORS.md) plugins.
+Reference the documentation on how to create these plugins written in Go.
+
+_For listed [external plugins](/EXTERNAL_PLUGINS.md), the author of the external plugin is also responsible for the maintenance
+and feature development of external plugins._
+
+1. Write your Telegraf plugin.  Depending on the plugin, follow the guidelines on how to create the plugin itself using InfluxData's best practices:
+   - [Input Plugins](/docs/INPUTS.md)
+   - [Processor Plugins](/docs/PROCESSORS.md)
+   - [Aggregator Plugins](/docs/AGGREGATORS.md)
+   - [Output Plugins](/docs/OUTPUTS.md)
+2. If your plugin is written in Go, include the steps for the [Execd Go Shim](/plugins/common/shim#steps-to-build-and-run-your-plugin)
+  1. Move the project to an external repo, it's recommended to preserve the path
+  structure, (but not strictly necessary). eg if your plugin was at
```

Comment:
same edits as other file

Reply:
I must've not pushed my latest edits before I got review for this. Sorry about the confusion! I had cleaned up all this wording and stuff.